packages/components/src/utils/checkbox-radio-button-wrapper/checkbox-radio-button-wrapper-utils.ts

```diff
@@ -0,0 +1,12 @@
+export const changeHandler = ({ target }: Event): void =>
```

Comment:
I think you could type `target` like this and wouldn't need to cast it twice.
```suggestion
export const changeHandler = ({ target }: Event & { target: HTMLInputElement }): void =>
```